src/plugins/saved_objects_management/server/routes/find.ts

```diff
@@ -90,9 +89,6 @@ export const registerFindRoute = (
         saved_objects: savedObjects.map((so) => {
           const obj = injectMetaAttributes(so, managementService);
           const result = { ...obj, attributes: {} as Record<string, unknown> };
-          for (const field of includedFields) {
```

Comment:
Removes handling the 'fields' option in the query since we don't want to explicitly expose attributes in the response